src/components/LoginForm.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Globalstyles from "../Globalstyles";
+import SignupButton from "../components/SignupButton";
+import TextFieldContainer from "../components/TextFieldContainer";
+import styled from "styled-components";
+
+const LoginFormStyled = styled.div`
```

Comment:
I would call this `LoginFormWrapper` because it is not a styled `LoginForm` component